package.json

```diff
@@ -6,33 +6,28 @@
   "module": "src/index.js",
   "scripts": {
     "fix:eslint": "eslint --fix --quiet src/**/**/*.js",
-    "fix:stylelint": "stylelint \"src/**/*.scss\" --cache --fix",
     "fix": "yarn fix:eslint",
     "test": "yarn test:lint-js && yarn test:unit",
-    "test:lint-css:watch": "onchange \"src/**/**/*.scss\" \"src/**/**/*.scss\" -- yarn test:lint-css",
-    "test:lint-css": "stylelint \"src/**/*.scss\" --cache",
     "test:lint-js": "eslint --cache --quiet src/**/**/*.js",
     "test:lint-js:watch": "onchange \"src/**/**/*.js\" -- yarn test:lint-js",
     "test:unit:verbose": "jest --coverage --no-cache",
     "test:unit": "yarn test:unit:verbose --silent",
     "test:unit:watch": "yarn test:unit:output -- --watch",
-    "test:unit:output": "jest --json --outputFile=jest-test-results.json --no-cache --silent",
+    "test:unit:output":
+      "jest --json --outputFile=jest-test-results.json --no-cache --silent",
     "prepush": "yarn test",
     "start": "yarn test:unit:output && start-storybook -p 6006",
     "prebuild": "yarn test:unit:output",
-    "build": "yarn build:stylesheet && yarn build:global-styles && yarn build:copy-global-styles && yarn build:storybook",
+    "build":
+      "yarn build:stylesheet && yarn build:global-styles && yarn build:copy-global-styles && yarn build:storybook",
     "build:storybook": "build-storybook -c .storybook -o .out",
-    "build:stylesheet": "mkdir -p dist && BABEL_ENV=static node build/stylesheet > dist/circuit-ui.css",
-    "build:global-styles": "mkdir -p dist && ./build/global-styles > dist/circuit-ui-global.css",
+    "build:stylesheet":
+      "mkdir -p dist && BABEL_ENV=static node build/stylesheet > dist/circuit-ui.css",
+    "build:global-styles":
+      "mkdir -p dist && ./build/global-styles > dist/circuit-ui-global.css",
     "build:copy-global-styles": "cp -f dist/circuit-ui-global.css .storybook",
     "deploy": "yarn build && gh-pages -d .out"
   },
-  "lint-staged": {
-    "*.js": [
-      "yarn fix:prettier",
-      "git add"
-    ]
-  },
```

Comment:
Why no more fixing code before pushing? Or is there a way to do this correctly with partially staged files?

Reply:
The code is still checked with eslint (which also runs prettier) before pushing, see [here](https://github.com/sumup/circuit-ui/pull/131/files/5b673b070e0ee89185d12c576c6aa1771b79e27e#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R18).
lint-staged was used for the precommit hook which we deprecated because it couldn't handle partially staged files.